documentation/source/reference.rst

```diff
@@ -964,7 +964,7 @@ Predefined Leaf Frames
    :operations:
 
       - :gf:`high-level-type` returns `<byte>
-        https://opendylan.org/library-reference/common-dylan/byte-vector.html#common-dylan:byte-vector:[byte]`_.
+        <https://opendylan.org/library-reference/common-dylan/byte-vector.html#common-dylan:byte-vector:[byte]>`_.
```

Comment:
I think this can just be ```returns :class:`<byte>`.```  Not sure though...it might depend on the InterSphinx extension being configured correctly and I'm not sure if I ever set that up.

That's just an FYI. We can worry about it later.